packages/tools/client-debugger/client-debugger-view/src/components/AudienceView.tsx

```diff
@@ -87,158 +86,92 @@ export function AudienceView(props: AudienceViewProps): React.ReactElement {
 		return <Waiting label="Waiting for Audience data." />;
 	}
 
+	// TODO: Determine if myClientMetaData is necessary
 	const myClientMetadata = audienceData.audienceState.find(
 		(audience) => audience.clientId === audienceData.clientId,
 	)?.client;
 
+	const audienceStateItems = AudienceStateDataFilter(
+		audienceData.audienceState,
+		myClientMetadata,
+	);
+	const audienceHistoryItems = AudienceHistoryDataFilter(audienceData.audienceHistory).reverse();
+
 	return (
-		<Stack
-			styles={{
-				root: {
-					height: "100%",
-				},
-			}}
-		>
-			<StackItem>
-				<h3>Audience members: ({audienceData.audienceState.length})</h3>
-				<MembersView
-					audience={audienceData.audienceState}
-					myClientId={audienceData.clientId}
-					myClientConnection={myClientMetadata}
-				/>
-			</StackItem>
-			<StackItem>
-				<h3>History</h3>
-				<HistoryView history={audienceData.audienceHistory} />
-			</StackItem>
-		</Stack>
+		<>
+			<Divider appearance="brand"> Audience State </Divider>
+			<AudienceStateTable audienceStateItems={audienceStateItems} />
+			<Divider appearance="brand"> Audience History </Divider>
+			<AudienceHistoryTable audienceHistoryItems={audienceHistoryItems} />
+		</>
 	);
 }
 
 /**
- * {@link MembersView} input props.
+ * Filtered audience state data for {@link AudienceStateDataFilter}
  */
-interface MembersViewProps {
-	/**
-	 * The current audience
-	 */
-	audience: AudienceClientMetadata[];
-
-	/**
-	 * My client ID, if the Container is connected.
-	 */
-	myClientId: string | undefined;
-
-	/**
-	 * My client connection data, if the Container is connected.
-	 */
+export interface FilteredAudienceStateData {
+	clientId: string;
+	userId: string;
+	mode: string;
+	scopes: string[];
 	myClientConnection: IClient | undefined;
 }
 
 /**
- * Displays a list of current audience members and their metadata.
+ * Removes unncessary data in audienceData.audienceState
  */
-function MembersView(props: MembersViewProps): React.ReactElement {
-	const { audience, myClientId, myClientConnection } = props;
-
-	const transformedAudience = combineMembersWithMultipleConnections(audience);
-
-	const memberViews: React.ReactElement[] = [];
-	for (const member of transformedAudience.values()) {
-		memberViews.push(
-			<StackItem key={member.userId}>
-				<AudienceMemberView
-					audienceMember={member}
-					myClientId={myClientId}
-					myClientConnection={myClientConnection}
-				/>
-			</StackItem>,
-		);
-	}
-
-	return <Stack>{memberViews}</Stack>;
+function AudienceStateDataFilter(
+	audienceStateData: AudienceClientMetadata[],
+	myClientConnection: IClient | undefined,
+): FilteredAudienceStateData[] {
```

Comment:
I'd rename this, the `AudienceHistoryDataFilter`, and the two related interfaces, since they're not doing any filtering.

Reply:
Also functions should be in `camelCase`

Reply:
I believe the two helper functions serve the general purpose of "filtering" unnecessary data from the `audienceData.audienceState` and `audienceData.audienceHistory`.

At first, the two JavaScript objects contain unnecessary metadata which is irrelevant to the information rendered for the `AudienceView` so I think the `fooDataFilter` is pertinent for the naming?

Reply:
I'd call that transformation more than filtering though; we're changing the shape of the data to something more suited to the task/component at hand, but we're not skipping any entries from the original list. "Filter" has the connotation of choosing which entries in a list or table should or shouldn't be considered for display or some next step (no better example than JS arrays' `filter()` method 😄). I don't know that these particular tables will ever need filtering like that, but I'd still recommend we don't overload the term.